Header header
uint8[] distance
uint8[] intensity
uint16[] rotation
uint16 scan_row
uint16 scan_column
